src/components/Composer.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import { push, ref as databaseRef, set } from "firebase/database";
+import {
+  getDownloadURL,
+  ref as storageRef,
+  uploadBytes,
+} from "firebase/storage";
+import { database, storage } from "../firebase";
+
+// Save the Firebase message folder name as a constant to avoid bugs due to misspelling
+const MESSAGE_FOLDER_NAME = "messages";
```

Comment:
I know the starter code calls it folder, but let's rename this to key, as these represent a reference to the keys in our key/value database
```suggestion
const MESSAGES_KEY = "messages";
```